src/main/resources/institutional-config.properties

```diff
@@ -117,6 +117,7 @@ cynergy.cuwal.useHttpHeaders=true
 cynergy.antivirus.service=antiVirusService
 cynergy.encryption.busEncryption=true
 cynergy.keystore.reloadSeconds=600
+rice.krad.jpa.kim.eclipselink.descriptor.customizer.org.kuali.rice.kim.impl.identity.entity.EntityBo=edu.cornell.cynergy.kim.impl.identity.entity.EntityBoDescriptorCustomizer
```

Comment:
I'm curious why we need to do this.  It looks like this same value is being set in cynergy-server-defaults-config.xml .  I don't see rice.krad.jpa.kim.eclipselink.descriptor.customizer.org.kuali.rice.kim.document.IdentityManagementRoleDocument reset here.

Reply:
We currently don't have a common institutional config file that is shared by both Cynergy and KFS, so I had to duplicate this property on both the standalone Cynergy and KFS sides. If we have more properties like this in the future, then we could look into adding such a file to cornell_customizations.

The IdentityManagementRoleDocument property is not on the KFS side since that's a standalone-Rice document that doesn't need its metadata customized in KFS. I'm adding this EntityBo property on both sides in case KFS needs to perform bounded searches on person data (since "Embedded" KIM run mode may cause it to run the query on the KFS side).

Reply:
thanks, that makes sense